_i18n/en.yml

```diff
@@ -351,8 +351,8 @@ help_pages:
         {: .number-list}
 
       ## Related articles
-      * [Authentication options](#)
-      * [Verify your identity](#)
+      * [Authentication options](site.baseurl/help/get-started/authentication-options/)
+      * [Verify your identity](site.baseurl/help/verify-your-identity/)
```

Comment:
whooops I independently took a stab at these in https://github.com/18F/identity-site/pull/545 and now these conflict 😬